.cirrus/tasks.yml

```diff
@@ -92,13 +92,15 @@ verify_gcp_app_template: &VERIFY_GCP_APP_TEMPLATE
   only_if: $TRIGGER == "PUSH_STAGING_DEPLOYER" || $BRANCH == "master"
   env:
     BASE_FOLDER: "/root/.gcp/cache"
-    PATH: "${BASE_FOLDER}/bin:${BASE_FOLDER}/google-cloud-sdk/bin:${PATH}"
+    PATH: "${BASE_FOLDER}:${BASE_FOLDER}/google-cloud-sdk/bin:${PATH}"
```

Comment:
i struggle understanding why we need this PATH variable at all considering we already modify it in the docker image ?

Reply:
As said above, we don't use the dockerfile here.